scripts/thrash_api.py

```diff
@@ -0,0 +1,130 @@
+# Authored by Bruce Martin
+# https://czi-sci.slack.com/archives/C023Q1APASK/p1667438044981199?thread_ts=1667431712.857239&cid=C023Q1APASK
```

Comment:
Since Slack messages will eventually expire (and this is a public repo), I think we should replace this link with a brief description of what this script is for

Reply:
Great idea! Just added a comment 👌 I kept the slack link because our public channels retain messages indefinitely unlike our DMs, so that way CZI members can gain more context. What do you think?

Thank you!

Reply:
I think it's just 1 year for public?

Reply:
@nayib-jose-gloria PTAL again thank you!

Reply:
I don't think anything was added since my last review @tihuan--did you push your local commits?

Reply:
Oh gosh sorry you right lol Just pushed!

@atolopko-czi Oh! I thought it was permanent 🤦 You're right that I'm unable to find any messages older than a year. Thanks for the headsup!

Maybe I'll add a comment to remove the Slack link once it reaches a year old?

Reply:
Added extra comment!

> # Note: The Slack link will expire around 11/2/2023 or whatever CZI's Slack retention policy entails